sonarqube/content.md

```diff
@@ -21,219 +21,37 @@ ulimit -n 65536
 ulimit -u 4096
 ```
 
-## Get Started in Two Minutes Guide
-
-/!\ This section shows you how to quickly run a demo instance. When you are ready to move to a more sustainable setup, take some time to read the **Configuration** section below.
-
-Start the server by running:
-
-```console
-$ docker run -d --name sonarqube -p 9000:9000 %%IMAGE%%
-```
-
-By default you can login as `admin` with password `admin`, see [authentication documentation](https://docs.sonarqube.org/latest/instance-administration/security/).
-
-To analyze a Maven project:
-
-```console
-# On Linux:
-$ mvn sonar:sonar
-
-# With boot2docker:
-$ mvn sonar:sonar -Dsonar.host.url=http://$(boot2docker ip):9000
-```
-
-To analyze other kinds of projects and for more details see [Analyzing Source Code documentation](https://redirect.sonarsource.com/doc/analyzing-source-code.html).
-
 ## Configuration
 
 ### Database
 
 By default, the image will use an embedded H2 database that is not suited for production.
 
-> Warning: Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
+> **Warning:** Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
 
-Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
+Set up a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Use bind-mounted folders or volumes
+### Use bind-mounted volumes
 
-The images contain the SonarQube installation folders at `/opt/sonarqube`. You will need to bind them to your host to override selected files or directories :
+The images contain the SonarQube installation volumes at `/opt/sonarqube`. You will need to bind them to your host to override select files or directories:
```

Comment:
Is this correct?
> to override select files or directories

Reply:
@michal-duda-sonarsource i'm not sure if this language should be removed

Reply:
We don't want to recommend bind-mounts for new images. In fact I would remove every occurrence of `bind-mounted` to not mislead people about our intention (especially from the name of the whole section). We had a choice between `bind-mounts` vs `volumes` and we went with `volumes` and that's the term we should use to be clear and consistent with docker docs.

As for this sentence, personally I don't understand it. I would rephrase it to: `We recommend creating volumes to store SonarQube data that you want to keep when you update to newer SonarQube version or switch to a higher SonarQube edition. The volumes should be used for the following directories in the container: ...`